exporter/elasticsearchexporter/README.md

```diff
@@ -118,33 +118,38 @@ Using the common `batcher` functionality provides several benefits over the defa
 
 ### Elasticsearch document routing
 
-Telemetry data will be written to signal specific data streams by default:
-logs to `logs-generic-default`, metrics to `metrics-generic-default`, and traces to `traces-generic-default`.
+Documents are routed to the target index / data stream dynamically in the following order. The first routing mode that applies will be used.
+1. "Static mode": To `logs_index` (for log records), `metrics_index` (for data points) and `traces_index` (for spans and span events) if they are configured.
+2. "Index attribute mode": To index name in `elasticsearch.index` attribute (precedence: log record / data point / span / span event attribute > scope attribute > resource attribute) if the attribute exists.
```

Comment:
[to reviewer] I took the liberty to use `elasticsearch.index` instead of `elasticsearch._index` for consistency, as we don't use underscore as a prefix to the last part of attribute name.